validation/validation.go

```diff
@@ -0,0 +1,72 @@
+package validation
+
+import (
+	"strconv"
+
+	"fmt"
```

Comment:
`go fmt` doesn't like this (move `fmt` up to line 4 before `strconv`).  Sorry, our concourse instance doesn't seem to have the fissile pipeline public… I'll check if we can fix that.